src/redux/actions.js

```diff
@@ -0,0 +1,11 @@
+export const ACTION = "ACTION";
+export const takeAction = () => async (dispatch, getState) => {
+  // const API = recieveAPIHere()
+  dispatch({
+    type: ACTION,
+    //    payload:
+    // API or else
+    // 0,
+    count: 1,
```

Comment:
This action always uses `1` as count. This does not sounds right. Shouldn't it be based on previous count?
It looks like it is not used in the reducer.